findMin.java

```diff
@@ -0,0 +1,31 @@
+class Solution {
+    public int findMin(int[] nums) {
+        int low = 0;
+        int high = nums.length -1;
+
+        while(low<high){
+        int mid = low + (high - low)/2;
+        if(low == high){
```

Comment:
Do you think that you will ever reach to this condition? Inside while loop?

Reply:
No

Reply:
Modified it. Thanks.